media_webdam.routing.yml

```diff
@@ -5,3 +5,11 @@ media_webdam.config:
     _title: 'Webdam Configuration'
   requirements:
     _permission: 'administer site configuration'
+
+media_webdam.webdam_upload:
+  path: '/admin/content/media/webdam'
+  defaults:
+    _form: '\Drupal\media_webdam\Form\WebdamUpload'
+    _title: 'Webdam Upload'
+  requirements:
+    _permission: 'administer site configuration'
```

Comment:
We'll need to change this permission later, but it's okay for now.

Reply:
ok. Will add a #TODO comment as a reminder.